enigma-server/docs/protocol.md

```diff
@@ -278,9 +286,10 @@ struct SyncMappingsS2CPacket {
 }
 struct MappingNode {
     NoParentEntry obf_entry;
-    boolean is_named;
```

Comment:
as far as I can tell this value is never written or read in `SyncMappingsS2CPacket#writeEntryTreeNode` and `SyncMappingsS2CPacket#readEntryTreeNode`. I've removed it since it doesn't seem to exist, but even if I'm wrong about this being outdated it would be obsoleted by including the token type and should be removed anyway.